docs/_snippets/checkbox-story.md

```diff
@@ -1,6 +1,4 @@
-```mdx renderer="common" language="mdx"
-{/* Checkbox.mdx */}
-
+```mdx filename="Checkbox.mdx" renderer="common" language="mdx"
 import { Canvas, Meta } from '@storybook/blocks';
 
 import * as CheckboxStories from './Checkbox.stories';
```

Comment:
This import needs to be `./Checkbox.stories.svelte` to work. I'm not sure how to achieve that here.